PolyDeploy/Components/APIUserManager.cs

```diff
@@ -9,13 +9,13 @@ internal static class APIUserManager
     {
         private static APIUserDataController APIUserDC = new APIUserDataController();
 
-        public static APIUser Create(string name)
+        public static APIUser Create(string name, bool bypass)
```

Comment:
Can we retain `Create(string name)` and have it call `Create(string name, book bypass)` with bypass set to false?

Just thinking ahead if new ways to create an APIUser are introduced, the default value for bypass should definitely be false.

Thoughts?

Reply:
Yes, this would definitely be better

Reply:
I've pushed this change